examples/version-migration/separate-container/tests/separateContainer.test.ts

```diff
@@ -79,17 +79,15 @@ describe("separate-container migration", () => {
 
 			await migrationP;
 
-			// Validate the migration status shows "two" after the migration
-			const leftContainsTwo = await page.evaluate(() => {
+			// After migration, the view should update.  Wait and confirm the migration status shows "two".
+			await page.waitForFunction(() => {
 				const migrationStatusElements = document.querySelectorAll(".migration-status");
 				return migrationStatusElements[0]?.textContent?.includes("two") === true;
 			});
-			const rightContainsTwo = await page.evaluate(() => {
+			await page.waitForFunction(() => {
 				const migrationStatusElements = document.querySelectorAll(".migration-status");
 				return migrationStatusElements[1]?.textContent?.includes("two") === true;
 			});
-			expect(leftContainsTwo).toEqual(true);
```

Comment:
Why are you removing the asserts?

Reply:
If waitForFunction() doesn't eventually get a "true" then the test will fail due to timeout.  Basically - if we get past the awaits then the assertion is true.